src/core/init.ts

```diff
@@ -439,7 +439,14 @@ export class InitCommand {
   }
 
   private async configureAITools(projectPath: string, openspecDir: string, toolIds: string[]): Promise<void> {
-    for (const toolId of toolIds) {
+    const resolvedToolIds = new Set(toolIds);
+
+    if (resolvedToolIds.has('opencode')) {
+      // OpenCode relies on the shared AGENTS.md instructions alongside slash commands
+      resolvedToolIds.add('agents');
```

Comment:
As discussed, if we can get away with just using the slash commands, would be ideal!